### PR TITLE
Adding AssociationsDebugger plugin

### PR DESCRIPTION
**Pull request by zunnu**
https://github.com/zunnu/associations-debugger

CakePHP 4 version will be made in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Additional lists you might find useful:
 - [Psa/FixtureCheck plugin](https://github.com/World-Architects/cakephp-fixture-check) - A plugin to help detect mismatches in live DB and fixtures in order to make fixture based tests more reliable and deployments safer.
 - :strawberry: [Sentry plugin](https://github.com/Connehito/cake-sentry) A plugin to seamlessly integrate Sentry with CakePHP for errors and exceptions.
 - :strawberry: [Setup plugin](https://github.com/dereuromark/cakephp-setup) - A lightweight setup plugin containing debugging and maintenance tools.
+- [AssociationsDebugger plugin](https://github.com/zunnu/associations-debugger) - A plugin that draws your CakePHP associations as diagram.
```
Comment (ADmad) [inline README.md:146]: Please follow alphabetical order.

Comment (zunnu) [reply README.md]: Updated

**Review by ADmad — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -136,6 +136,7 @@ Additional lists you might find useful:
 *Plugins for debugging.*
 
 - [Airbrake plugin](https://github.com/chrisShick/AirbrakeCake) A plugin to seamlessly integrate Airbrake with CakePHP for errors and exceptions.
+- [AssociationsDebugger plugin](https://github.com/zunnu/associations-debugger) - A plugin that draws your CakePHP associations as diagram.
```
Comment (dereuromark) [inline README.md:139]:
```suggestion
- [AssociationsDebugger plugin](https://github.com/zunnu/associations-debugger) - A plugin that draws your model associations as diagram.
```

Comment (zunnu) [reply README.md:139]: Looks good

**Pull request merged**